src/Arcus.Scripting.ApiManagement/Scripts/Backup-AzApiManagementService.ps1

```diff
@@ -2,13 +2,19 @@ param(
     [Parameter(Mandatory = $true)][string] $ResourceGroupName = $(throw "Resource group name is required"),
     [Parameter(Mandatory = $true)][string] $StorageAccountResourceGroupName = $(throw = "Resource group for storage account is required"),
     [Parameter(Mandatory = $true)][string] $StorageAccountName = $(throw "Storage account name is required"),
-    [Parameter(Mandatory = $true)][string] $ServiceName = $(throw "API managgement service name is required"),
+    [Parameter(Mandatory = $true)][string] $ServiceName = $(throw "API management service name is required"),
     [Parameter(Mandatory = $true)][string] $ContainerName = $(throw "Name of the target blob container is required"),
-    [Parameter(Mandatory = $false)][string]  $BlobName = $null,
+    [Parameter(Mandatory = $true)][string][ValidateSet('SystemAssignedManagedIdentity', 'UserAssignedManagedIdentity')] $AccessType = $(throw "The access type is required"),
```

Comment:
Comment is a bit late to the party 😜 I was wondering if we could change the parameter type here to something like an enumeration, so that we could o `-AccessType SystemAssignedManagedIdentity`. I think the intelisense will also then be able to help, I believe.

Reply:
Better late than never 😉 

I'm not really sure what having an enum would provide above having the `ValidateSet`, as I understand this already has tab completion: https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_functions_argument_completion?view=powershell-7.4#validateset-attribute.
> The ValidateSet attribute specifies a set of valid values for a parameter or variable and enables tab completion. 

We also use the `ValidateSet` in several other scripts, might be better to keep this all in line? 
What do you think?

Reply:
Aha, okay, thx for the context. Yes, then it is better to use this approach. 👍